### ISSUE TITLE
Can you provide the code for gradient descent?

### ISSUE THREAD

**Issue by i4yyds** [Dawars/auto_deeplab-pytorch#1]
how to implement trainning? thanks.

**Comment by ghost** [comment]
@Dawars
